include/NovelRT.h

```diff
@@ -187,6 +173,9 @@
   #include "NovelRT/Animation/SpriteAnimatorState.h"
   #include "NovelRT/Animation/SpriteAnimator.h"
 
+// Audio types
+ #include "NovelRT/Audio/Audio.h"
```

Comment:
Spacing/indentation looks a little wonky here?

Reply:
Haven't applied clang-format yet, and I may have spaced a little weird 😅 still getting used to CLion

Reply:
Manually resolved for now